src/WebView.ios.tsx

```diff
@@ -182,6 +184,7 @@ const WebViewComponent = forwardRef<{}, IOSWebViewProps>(({
       onLoadingStart={onLoadingStart}
       onHttpError={onHttpError}
       onMessage={onMessage}
+      onOpenWindow={onOpenWindowProp && onOpenWindow}
```

Comment:
why isn't `onOpenWindow={onOpenWindow}` enough?

Reply:
At some point I had to use this code so the iOS code would be able to see a null value.

I'm pretty sure I needed this on my initial code based on version `11.18.2`. Cf this PR: https://github.com/cozy/react-native-webview/pull/2 where `onOpenWindow` is always defined by construction:
```js
onOpenWindow = (event: WebViewOpenWindowEvent) => {
    const { onOpenWindow } = this.props;
    if (onOpenWindow) {
      onOpenWindow(event);
    }
  };
```

But since I rebased on master, I adapted the code to the new architecture and now the prop is declared
```
const onOpenWindow = useCallback((event: WebViewOpenWindowEvent) => {
    onOpenWindowProp?.(event);
  }, [onOpenWindowProp]);
```

But I don't remember if I tested to remove it after that refactoring 🤔 
(damn brain that cannot remember details from 3weeks ago)

Reply:
I just did the test, I confirm that `onOpenWindow={onOpenWindow}` does not detect correctly if the event is defined or not, `_onOpenWindow` is alwary truthy in `RNCWebView.m`.

With `onOpenWindow={onOpenWindowProp && onOpenWindow}` the detection is done correctly.

Reply:
Hmmm. That makes sense.... but then confuses me because I would expect that to mean `messagingEnabled` is always true also??

Uh. Hm. Yeah. That appears to be exactly what's happening. So that seems like a bug. Dang.

Thanks for helping identify that.

Reply:
(FYI @Titozzz ... I'm not sure if there's an observable regression that would result from `messagingEnabled` always being true, but it does seem unintended)

Reply:
(opened #2666 based on this conversation)